layouts/partials/contact_foot_en.html

```diff
@@ -0,0 +1,41 @@
+<br />
+<br />
+    <div itemprop="address" itemscope itemtype="http://schema.org/PostalAddress">
+        <div class="halfPanel">
+		 <h2>E-mail Contacts</h2>
+
+        <dl>
+                <dt>General information</dt>
+                <dd><a href="mailto:info@enectiva.cz" itemprop="email">info@enectiva.cz</a></dd>
+
+                <dt>Technical support</dt>
+                <dd><a href="mailto:podpora@enectiva.cz" itemprop="email">podpora@enectiva.cz</a></dd>
+
+                <dt>Business issue</dt>
+                <dd><a href="mailto:obchod@enectiva.cz" itemprop="email">obchod@enectiva.cz</a></dd>
+
+                <dt>Bussiness partnership</dt>
+                <dd><a href="mailto:partner@enectiva.cz" itemprop="email">partner@enectiva.cz</a></dd>
+
+        </dl>
+        </div>
+
+        <div class="halfPanel">
+          <h2>Phone Contacts</h2>
+          <ul>
+            <li>Phone : <span itemprop="telephone">+420 222 766 950</span></li>
+            <li>Phone : <span itemprop="telephone">+420 222 766 951</span></li>
+            <li>Mobile : <span itemprop="telephone">+420 732 916 385</span></li>
+            <li>Fax : <span itemprop="faxNumber">+420 222 766 959</span></li>
```

Comment:
I'm not sure I understand what you mean.

In Czech, you don't place a space in front of a semicolon. Same in English (there might be some case whern you do but it's not this). I don't know about the other languages. I'll take your word for Spanish, but I just don't know for the rest. I know that in French, they put spaces in front of question marks, so spaces in fornt of semicolons seem plausible. So, please, try to google it and default to no space if you don't find anything.